.github/workflows/ci.yaml

```diff
@@ -138,22 +142,3 @@ jobs:
       run: make helm-lint
     - name: Helm Unit Tests
       run: make helm-test
-
-  # Action that will automatically comment on PRs based on changes
-  pr-comments:
```

Comment:
Moved to a separate pr-comments workflow since we need `on: pull_request_target` so we can comment on PRs across forks.